Gemfile.lock

```diff
@@ -0,0 +1,18 @@
+GEM
+  remote: https://rubygems.org/
+  specs:
+    rack (1.6.4)
+    rack-cors (0.4.0)
+    roda (2.18.0)
+      rack
+
+PLATFORMS
+  ruby
```

Comment:
Good point, pushed an additional commit once it has been built with the correct ruby version. If any further changes are needed please feel free to add them.